graphql2/models_gen.go

```diff
@@ -485,6 +485,18 @@ type SlackChannelSearchOptions struct {
 	Omit   []string `json:"omit"`
 }
 
+type SlackUserGroupConnection struct {
+	Nodes    []slack.UserGroup `json:"nodes"`
+	PageInfo *PageInfo         `json:"pageInfo"`
```

Comment:
Target standards for json is to use snake case, not camelCase.  Is this json for our app, or how we interact with Slack?

https://apiplatform-docsite.prod.target.com/docs/customer-engagement/api-design-standards/#use-snake_case-not-camelcase

Reply:
This is generated code and based on the schema